### PR TITLE
Pull form fields into a namespaced 'fields' property.

### PR DESCRIPTION
**Pull request by garethbowen**
Issue: medic/medic-webapp#957

### CODE REVIEW

File: tests/migrations/namespace-form-fields.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  var getConfig = sinon.stub(config, 'get').returns(forms);
+  var insert = sinon.stub(db.medic, 'insert').callsArgWith(1, null, null);
+  migration.run(function(err) {
+    test.equals(err, undefined);
+    test.equals(getView.callCount, 1);
+    test.equals(getDoc.callCount, 1);
+    test.equals(getConfig.callCount, 1);
+    test.equals(insert.callCount, 1);
+    test.same(insert.args[0][0].fields, expected);
+    test.equals(insert.args[0][0].last_menstrual_period, null);
```
Comment (mandric) [inline tests/migrations/namespace-form-fields.js]: should these fields be null or undefined now?

Comment (garethbowen) [inline tests/migrations/namespace-form-fields.js]: `equals` tests with `==` so this assertion passes with either `null` or `undefined`. I could use `same` to be explicit about it if you'd prefer?

Comment (mandric) [inline tests/migrations/namespace-form-fields.js]: Well there is a difference, null will appear on the doc but undefined won't.

Comment (garethbowen) [inline tests/migrations/namespace-form-fields.js]: Right. They'll be undefined now as I `delete` them off the doc.

Comment (mandric) [inline tests/migrations/namespace-form-fields.js]: If we don't want them on the doc then we should probably assert they are undefined?

**Pull request merged**